app/src/main/java/com/todoist_android/ui/home/BottomSheetEditTaskFragment.kt

```diff
@@ -48,8 +48,8 @@ class BottomSheetEditTaskFragment : BottomSheetDialogFragment() {
     private var selectedReminderTime: String? = null
     private var dueDate: String? = null
     private var newDueDate: String? = null
-    private var reminder: String? = null
-    private var status = "created"
+    private var taskReminder: String? = null
```

Comment:
@Njumbi have we notified backend on this `reminder` parameter... was going through the endpoints doc and didn't see it documented there

Reply:
I had asked them to add it, 
I will also remind them.